spec/components/create_log_actions_component_spec.rb

```diff
@@ -37,13 +37,23 @@
         expect(component.create_button_href).to eq("/lettings-logs")
       end
 
-      it "returns upload button copy" do
-        expect(component.upload_button_copy).to eq("Upload lettings logs in bulk")
+      it "does not show the upload button" do
```

Comment:
Why did this change?

Reply:
We don't show the upload lettings logs in bulk button on that page. I changed it within a previous ticket #2629, just added a test for that. We don't need to return the upload button copy or upload button href one this page, for support users. We do for other users.